src/main/scala/com/gu/scanamo/ScanamoFree.scala

```diff
@@ -14,18 +14,34 @@ object ScanamoFree {
   import cats.syntax.traverse._
   import collection.JavaConverters._
 
+  private val batchSize = 25
+
   def put[T](tableName: String)(item: T)(implicit f: DynamoFormat[T]): ScanamoOps[PutItemResult] =
     ScanamoOps.put(ScanamoPutRequest(tableName, f.write(item), None))
 
   def putAll[T](tableName: String)(items: Set[T])(implicit f: DynamoFormat[T]): ScanamoOps[List[BatchWriteItemResult]] =
-    items.grouped(25).toList.traverseU(batch =>
+    items.grouped(batchSize).toList.traverseU(batch =>
       ScanamoOps.batchWrite(
         new BatchWriteItemRequest().withRequestItems(Map(tableName -> batch.toList.map(i =>
           new WriteRequest().withPutRequest(new PutRequest().withItem(f.write(i).getM))
         ).asJava).asJava)
       )
     )
 
+  def deleteAll[T](tableName: String)(deleteItems: (Symbol, Set[T]))(implicit f: DynamoFormat[T]): ScanamoOps[List[BatchWriteItemResult]] = {
```

Comment:
I was trying `Set[UniqueKey[_]]` and ran across `UniqueKeys[_]` which I think is a better fit.  about to push an update.  Let me know what you think.